### PR TITLE
Add "Unspecified exception" section

### PR DESCRIPTION
**Pull request by fatkodima**

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -441,6 +441,20 @@ refute('rubocop-minitest'.kind_of?(String))
...
 
+=== Unspecified exception [[unspecified-exception]]
+
+Specify the exception being captured by `assert_raises`. This avoids false-positives
+when the raised exception is not the same you were expected.
+
+[source,ruby]
+----
+# bad
+assert_raises { do_something}
```
Comment (koic) [inline README.adoc:452]:
```suggestion
assert_raises { do_something }
```

File: README.adoc [hunk 2]
```diff
@@ -441,6 +441,20 @@ refute('rubocop-minitest'.kind_of?(String))
 refute_kind_of(String, 'rubocop-minitest')
 ----
 
+=== Unspecified exception [[unspecified-exception]]
+
+Specify the exception being captured by `assert_raises`. This avoids false-positives
+when the raised exception is not the same you were expected.
```
Comment (koic) [inline README.adoc:447]: It's a small wording difference, what about the following?

```suggestion
when the raised exception is not the same users were expected.
```

**Comment by fatkodima** [comment]
Updated.

**Comment by koic** [comment]
Thanks!

**Pull request merged**